internal/server/turn.go

```diff
@@ -54,20 +54,33 @@ func (s *SOMASServer) runOrgs() error {
 }
 
 // endOfTurn performs end of turn updates
+// TODO: organise order of end of turn actions
 func (s *SOMASServer) endOfTurn() error {
 	s.logf("start endOfTurn")
 	defer s.logf("finish endOfTurn")
 
-	err := s.runOrgsEndOfTurn()
-	if err != nil {
-		return errors.Errorf("Failed to run orgs end of turn: %v", err)
+	if err := s.runIIGOTax(); err != nil {
+		return errors.Errorf("Failed to put taxes into common pool at end of turn: %v", err)
 	}
 
-	err = s.runForage()
-	if err != nil {
+	if err := s.runIIFOEndOfTurn(); err != nil {
```

Comment:
IIFO and IITO do not currently have any end of turn action currently and I doubt they will in the future.
Would you mind deprecating these two functions, just delete them from here and there declaration in the iito and iifo files

Reply:
Sure, I can do that! Just to check then, are gifts actually given and received during the IITO session? I thought it was just like an agreement that was made but it actually happened at the end of the turn.

Reply:
With actions being deprecated it makes more sense to just give the clients the resources as soon as they make the agreement, I don't see any reason to wait till the end of turn.

Currently IITO doesn't update the client state but there is a PR with an overhaul that will include that soon.

Reply:
I think maybe we should check with the rest of design on this as I recall there being conversations about how it should appear that islands make decisions on what to do with the resources that they have concurrently. The purpose of this was to ensure that a decision on one action wasn't influenced by the outcome of another. This can be achieved by doing all actions that subtract from their pool before actions that add to their pool. 

Just to say, not sure that this reasoning is correct anymore but I remember this being decided on.

Reply:
I'll remove IIFO end of turn actions in the meantime

Reply:
Mm that does seem like a fair point. Alright we can just merge these changes then a revisit it later. Is that okay?

Reply:
sure! it's just a pr into @mpardalos ' branch